apps/logsupport_appdefs.cpp

```diff
@@ -0,0 +1,41 @@
+
+
+/*
+ * SRT - Secure, Reliable, Transport
```

Comment:
These extra blank lines could be removed.

Reply:
Sadly, quite a big part of the code in the script is to deal with spaces and indents. And it continues.

Reply:
If it is too hard to delete - no problem. Let it stay.